packages/server/.gitignore

```diff
@@ -1,2 +1 @@
-.env
```

Comment:
Why are we removing this from here and `services` if still have those files?

Reply:
cause now we have a `.env` in the root dir and I moved the ignore statement there